experiments/run_experiment.py

```diff
@@ -123,20 +131,49 @@ def initialize_recourse_method(
             print("Dataset: {}".format(data_name))
             print("Model type: {}".format(model_type))
 
-            mlmodel = MLModelCatalog(dataset, model_type, backend)
-
-            factuals = predict_negative_instances(mlmodel, dataset)
-            factuals = factuals.iloc[: args.number_of_samples]
-            factuals = factuals.reset_index(drop=True)
-
-            if rm == "dice":
-                mlmodel.use_pipeline = True
-
-            recourse_method = initialize_recourse_method(
-                rm, mlmodel, dataset, data_name, model_type, setup
-            )
-
-            df_benchmark = Benchmark(mlmodel, recourse_method, factuals).run_benchmark()
+            if "cem" in rm:
```

Comment:
If we need to have if/else can we do something like `if rm in session_models` and maintain a list of models (so far only cem in code)

Reply:
Also do we need to check for pytorch/tensorflow model?